torch/_dynamo/logging.py

```diff
@@ -1,21 +1,34 @@
+import dis
 import itertools
 import logging
 import os
+import types
+import typing
+
+from torch._guards import Guard
+from torch._logging import init_logs, loggable, register_log
+from torch.fx.graph_module import GraphModule
 
 from torch.hub import _Faketqdm, tqdm
 
-# logging level for dynamo generated graphs/bytecode/guards
-logging.CODE = 15
-logging.addLevelName(logging.CODE, "CODE")
+
+TORCHDYNAMO_LOG_NAME = "torch._dynamo"
+TORCHINDUCTOR_LOG_NAME = "torch._inductor"
+AOT_AUTOGRAD_LOG_NAME = "torch._functorch.aot_autograd"
```

Comment:
Handling AOTAutograd in torch._dynamo feels like a layering violation. cc @Chillee

Reply:
This was tricky, I wanted to setup all of the PT2 logs in one place. AOT is special since it can get called standalone, (I guess inductor can now too) but yeah, curious if you have ideas here.

Reply:
In my ideal world, it's not necessary to have `torch._dynamo.logging`; per component logging can be setup in their respective __init__.py (e.g., torch/_dynamo/__init__.py) directly

Reply:
so I don't do this as much now, but there is still this feeling of intertwinedness in some places (ie the testing, but not sure I care as much there)